src/pydvl/shapley/__init__.py

```diff
@@ -113,7 +141,7 @@ def compute_shapley_values(
     )
 
     if val_std is None:
-        df["data_value_std"] = np.nan
+        df["data_value_std"] = np.nan  # FIXME: why NaN? stddev of a constant RV is 0
```

Comment:
I used NaN here by force of habit and to make sure that the values are ignored when plotting the values (which isn't the case with 0). Still I think 0 makes more sense.